modules/ossm-understanding-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/understanding-ossm.adoc
+
+[id="ossm-understanding-architecture_{context}"]
+= Understanding the {ProductName} Architecture
```

Comment:
Gerunds for procedures only.

Reply:
Fixed and updated file name for module and in assembly.